### ISSUE TITLE
How to get postgresql error code

### ISSUE THREAD

**Issue by alexishevia** [robconery/massive-js#214]
On previous massive-js versions, the error object would bring info like:
```
{
    "code": "23505",
    "constraint": "events_pkey",
    "detail": "Key (id)=(test123) already exists.",
    "file": "nbtinsert.c",
    "length": 222,
    "line": "406",
    "name": "error",
    "routine": "_bt_check_unique",
    "schema": "public",
    "severity": "ERROR",
    "table": "events",
    "where": "SQL function \"createEvent\" statement 1"
}
```
So it was pretty easy for me to grab the error code and check against [PostgreSQL Error Codes](http://www.postgresql.org/docs/9.3/static/errcodes-appendix.html) to determine how I wanted to handle it.

I updated to massive-js 2.1.0, and now the error object is limited to returning a single message:
```
{ "message": "duplicate key value violates unique constraint \"events_pkey\"" }
```

Is there any way I can get access to the error code as before?

**Comment by robconery** [comment]
This was changed (I think) in a PR I just pulled in - try master? If not happy to have a PR for it :).

**Comment by alexishevia** [comment]
I get the same result with master. I'll try to figure out how to get the error code back and will send a PR if I can.

**Comment by diegoaguilar** [comment]
@alexishevia did u achieve it?

**Comment by alexishevia** [comment]
@diegoaguilar I found out the root cause. On commit 0b56fd5148591bdd15c98d8f0250b209b4770d4f we stopped bubbling up the error given by `db.query`, and instead we construct a new Error object. Since only the `message` attribute is attached to the Error object, all other info is lost.

I was able to get the error code back on this commit: https://github.com/alexishevia/massive-js/commit/a8842cde885cebd61ebad9e44a9be278bf36ee5d

I wanted to add a test before sending the PR, but haven't had time for that. Will probably submit it in a couple of days (unless you want to take over).
